### PR TITLE
[Issue-#126] Include a newline character before appending talisman info

### PR DESCRIPTION
**Pull request by anusha5695**
This is the PR for #126

### CODE REVIEW

File: global_install_scripts/install.bash [hunk 1]
```diff
@@ -228,6 +228,7 @@ function run() {
 		if [ -f $FILE_PATH ] && grep -q "TALISMAN_HOME" $FILE_PATH; then
 			if ! grep -q ">>> talisman >>>" $FILE_PATH; then
 				sed -i'-talisman.bak' '/TALISMAN_HOME/d' $FILE_PATH
+				echo "\n" >>${ENV_FILE}
```
Comment (svishwanath-tw) [inline global_install_scripts/install.bash]: Please change this and all subsequent occurrences to `echo -e "\n"`

Comment (anusha5695) [reply global_install_scripts/install.bash]: have changed. thanks

File: global_install_scripts/update_talisman.bash [hunk 2]
```diff
@@ -156,6 +156,7 @@ function run() {
 		if [ -f $FILE_PATH ] && grep -q "TALISMAN_HOME" $FILE_PATH; then
 			if ! grep -q ">>> talisman >>>" $FILE_PATH; then
 				sed -i'-talisman.bak' '/TALISMAN_HOME/d' $FILE_PATH
+				echo "\n" >>${ENV_FILE}
```
Comment (svishwanath-tw) [inline global_install_scripts/update_talisman.bash]: This one too please (echo -e)

Comment (anusha5695) [reply global_install_scripts/update_talisman.bash]: All done now

**Pull request merged**